### PR TITLE
Rune Scimitar/Armor Set

### PR DESCRIPTION
**Pull request by GermanSanta**
Too long have we gone without the classic rune scimmy. I wasn't too sure if i should make this a wizard weapon or not so for now i made it slightly stronger than a toolbox with a bleed chance. This bad boy can be purchased in Al's fun and games for the humble price of 25k. 

Going to lead editing open for this in case color/power/location contrasts too much.

Selling+rune+scimitar+25k6
WGkq4f0OQ1ufzWqoSBNtGw

### CODE REVIEW

**Comment by SonixApache** [comment]
hey look more dumb PRs

**Comment by calzilla1** [comment]
Something something throw wire

**Comment by SkowronX** [comment]
let them have fun sonix. also why not help them with their code?

**Comment by Lokiamis** [comment]
Gielinorstation when

**Comment by Dacendeth** [comment]
Hmmm a free 25 force weapon what could go wrong.

**Comment by Lokiamis** [comment]
I'm not opposed to a Runescape reference, but it doesn't make sense to put a dangerous sword in toy vending machine

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -3356,7 +3357,8 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/dice/loaded = 15,
 		/obj/item/weapon/dice/loaded/d20 = 15,
 		/obj/item/weapon/skull = 20,
-		/obj/item/weapon/storage/bag/wiz_cards/frog = 20
+		/obj/item/weapon/storage/bag/wiz_cards/frog = 20,
+		/obj/item/weapon/melee/rsscimmy = 25000
```
Comment (MadmanMartian) [inline code/game/machinery/vending.dm]: Rune scimitar has a base value of 25,600 according to the OSRS wiki.

https://oldschool.runescape.wiki/w/Rune_scimitar

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 2]
```diff
@@ -285,3 +285,20 @@
...
+/obj/item/weapon/melee/rsscimmy
+	name = "rune scimitar"
+	desc = "A vicious, curved sword."
+	icon_state = "rsscimmy"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	hitsound = 'sound/weapons/runescapeslash.ogg'
+	flags = FPRINT
+	siemens_coefficient = 1
+	sharpness = 1.5
+	sharpness_flags = SHARP_BLADE | SERRATED_BLADE | CHOPWOOD
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/swords_axes_etc.dm]: A scimitar is not serrated

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 3]
```diff
@@ -285,3 +285,20 @@
...
+
+/obj/item/weapon/melee/rsscimmy
+	name = "rune scimitar"
+	desc = "A vicious, curved sword."
+	icon_state = "rsscimmy"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	hitsound = 'sound/weapons/runescapeslash.ogg'
+	flags = FPRINT
+	siemens_coefficient = 1
+	sharpness = 1.5
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/swords_axes_etc.dm]: 187.5% chance of forcibly amputating a limb should the limbs health be greater than or equal to its max damage. Not an issue, just stating such

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 4]
```diff
@@ -285,3 +285,20 @@
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
 	item_state = "grey_sword"
 	force = 4
+
+/obj/item/weapon/melee/rsscimmy
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/swords_axes_etc.dm]: Why not have it be a subtype of /obj/item/weapon/sword?

**Comment by MadmanMartian** [comment]
Icons need some TLC

**Comment by Probe1** [comment]
I'd merge it but address all those things MadmanMartian said.  Especially the high damage and straight up ripped from google images icon

**Comment by MadmanMartian** [comment]
It's only been up 2 days, why would you speedmerge it?

**Comment by SonixApache** [comment]
Why would you merge it?

File: code/game/objects/items/weapons/swords_axes_etc.dm (hunk 2 above)
Comment (boy2mantwicethefam) [reply code/game/objects/items/weapons/swords_axes_etc.dm]: to be fair it's really blocky

**Comment by utilitas-** [comment]
@Dacendeth
```
/obj/item/weapon/storage/toolbox/syndicate
	name = "suspicious looking toolbox"
	force = 20
```
Easily accessible instances on Boxstation and most other maps found on the derelict: 3.
Deaths by syndie toolboxing: Basically null.
Fuckeddy fuck off.

Also, Sonix, please take some time off for yourself, you are having some broin problems.

**Comment by brndd** [comment]
If this has the same icons as the ones posted on Discord, I agree they need some work. The colour is/was off and the scimitar itself could do with a bit more detail on it IMO.

**Comment by SkowronX** [comment]
confrictu

**Comment by GermanSanta** [comment]
Massive Massive update this! 
Sprites = Given Real RS colors
Scimitar = lower damage, less force, only 1 in vendor
Armor Set located in Cargo Knight Crate
Open for changes and possibility to becoming a vault item
tC8NumPWRraNNGOrEDrsdw
download
Aum0Ek-bQq_au6OW4V5zHA
-xoK2TL0SPm63c7R5Iyi7g

**Comment by SonixApache** [comment]
*why*

**Comment by brndd** [comment]
Looking real good now and like I said on Discord, these would be perfect for a Runescape-themed asteroid vault or something.

**Comment by Kharkov72** [comment]
gonfligds haha

**Comment by datemononeko** [comment]
sonix when are you going to make the clown removal PR

**Comment by SonixApache** [comment]
i don't hate fun, just pointless "literally only exist to be a reference to something" reskins of other items

**Pull request closed without merging**